app/helpers/podcast_metrics_helper.rb

```diff
@@ -0,0 +1,96 @@
+module PodcastMetricsHelper
+  def line_chart_colors
+    ["#007EB2", "#FF9600", "#75BBE1", "#FFC107", "#6F42C1", "#DC3545", "#198754", "#D63384", "#20C997", "#555555"]
+  end
+
+  def parse_agent_data(agents)
+    agents.map do |a|
+      {
+        x: a.label,
+        y: a.count
+      }
+    end
+  end
+
+  def sum_rollups(rollups)
+    rollups.map { |r| r[:count] }.reduce(:+)
+  end
+
+  def interval_options(date_start, date_end)
+    day_count = (date_start.to_date..date_end.to_date).to_a.length
```

Comment:
a loose "guard" of sorts to discourage the user from selecting HOUR on really large ranges, or MONTH on small ranges. not a true guard since the user can still shift the dates. could detail this later.